sdk/communication/Azure.Communication.Chat/tests/ChatClients/ChatClientsLiveTest.cs

```diff
@@ -168,24 +162,27 @@ public void ThreadCGUD_MemberAUR_MessageGSU_NotificationT()
             Pageable<ChatMessage> messagesAfterOneDeleted = chatThreadClient.GetMessages();
             ChatMessage deletedChatMessage = messagesAfterOneDeleted.First(x => x.Id == messageId);
 
-            #region Snippet:Azure_Communication_Chat_Tests_E2E_GetMembers
-            Pageable<ChatThreadMember> chatThreadMembers = chatThreadClient.GetMembers();
-            #endregion Snippet:Azure_Communication_Chat_Tests_E2E_GetMembers
-            var chatThreadMembersCount = chatThreadMembers.Count();
-
-            var newMember = new ChatThreadMember(user4);
-            #region Snippet:Azure_Communication_Chat_Tests_E2E_AddMembers
-            chatThreadClient.AddMembers(members: new[] { newMember });
-            #endregion Snippet:Azure_Communication_Chat_Tests_E2E_AddMembers
-            Pageable<ChatThreadMember> chatThreadMembersAfterOneAdded = chatThreadClient.GetMembers();
-            var chatThreadMembersAfterOneAddedCount = chatThreadMembersAfterOneAdded.Count();
-
-            CommunicationUser memberToBeRemoved = user4; //Better name for the snippet
-            #region Snippet:Azure_Communication_Chat_Tests_E2E_RemoveMember
-            chatThreadClient.RemoveMember(user: memberToBeRemoved);
-            #endregion Snippet:Azure_Communication_Chat_Tests_E2E_RemoveMember
-            Pageable<ChatThreadMember> chatThreadMembersAfterOneDeleted = chatThreadClient.GetMembers();
-            var chatThreadMembersAfterOneDeletedCount = chatThreadMembersAfterOneDeleted.Count();
+            #region Snippet:Azure_Communication_Chat_Tests_E2E_GetParticipants
+            Pageable<ChatParticipant> chatParticipants = chatThreadClient.GetParticipants();
+            #endregion Snippet:Azure_Communication_Chat_Tests_E2E_GetParticipants
+            var chatParticipantsCount = chatParticipants.Count();
+
+            var newParticipant = new ChatParticipant(user4);
+            var newParticipant2 = new ChatParticipant(user5);
+            #region Snippet:Azure_Communication_Chat_Tests_E2E_AddParticipants
+            chatThreadClient.AddParticipants(participants: new[] { newParticipant });
+            #endregion Snippet:Azure_Communication_Chat_Tests_E2E_AddParticipants
+            chatThreadClient.AddParticipant(newParticipant2);
+
+            Pageable<ChatParticipant> chatParticipantsAfterTwoAdded = chatThreadClient.GetParticipants();
+            var chatParticipantsAfterTwoAddedCount = chatParticipantsAfterTwoAdded.Count();
+
+            CommunicationUser participantToBeRemoved = user4; //Better name for the snippet
```

Comment:
What is the comment about?

Reply:
It was added to make reviewers aware that it is just for cosmetic reasons on the snippet.
Will remove it now.